server/app.js

```diff
@@ -0,0 +1,28 @@
+require("dotenv").config();
```

Comment:
change to config({path: <path to our .env>}) as it can no longer be in root because the .gitignore is not in root